ui/src/app/metadata/named-references/named-reference-table/named-reference-table.component.html

```diff
@@ -48,7 +56,7 @@
         [selectAllEnabled]="selectAllEnabled"
         (valueChange)="handleSelectAll($event)"
         [totalCount]="getSelectAllCount()"
-        [totalPageCount]="items.length">
+        [totalPageCount]="0">
```

Comment:
Probably should display only one option in select, select 0 doesn't make sense.

<img width="292" alt="Screenshot 2023-06-22 at 17 29 19" src="https://github.com/wgu-opensource/osmt/assets/68391066/39498ad2-839e-453a-84d7-1c24ff972b78">

Reply:
I guess this was a typo, I dont recall hardcoding this.